tests/bson/bson-utcdatetime-int-size.phpt

```diff
@@ -1,5 +1,7 @@
 --TEST--
 Test for UTCDateTime and integer parsing
+--SKIPIF--
+<?php if (defined("HHVM_VERSION_ID")) exit("skip HHVM implements only an older version of DateTime"); ?>
```

Comment:
I've cloned this one, and adapted it for HHVM already. Looks like I had forgotten to commit it though, but I have now: https://github.com/mongodb/mongo-hhvm-driver/blob/master/tests/bson-utcdatetime-int-size.phpt